plutus-tx/src/PlutusTx/Show/TH.hs

```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE LambdaCase        #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module PlutusTx.Show.TH where
+
+import PlutusTx.Builtins
+import PlutusTx.Foldable
+import PlutusTx.List
+
+{- | Conversion of values to `BuiltinString`s. Unlike @GHC.Show.Show@, there is no
+ @showList@ method, because there is no `Show` instance for `Data.String.String`.
```

Comment:
I think I'm missing some context, but this still leaves me a bit mystified.

Reply:
I'm not sure what you mean; do you mean we need `showList`?

Reply:
I think the `showList` method in base is specifically for the `Show String` instance. Otherwise why not also have `showPair`, `showMaybe` or whatever.